include/picongpu/plugins/transitionRadiation/TransitionRadiation.kernel

```diff
@@ -138,7 +140,7 @@ namespace picongpu
                     int const numSuperCells = superCellsCount.productOfComponents();
 
                     // propagation distance for the particle bunch
-                    float_X const propagationDistance = parameters::foilPosition - globalOffset[1];
+                    float_X const propagationDistance = foilPosition - globalOffset[1];
```

Comment:
Just for my understanding, why do you subtract  `globalOffset[1]`?

Reply:
This took me some minutes to figure it out myself. I think, the idea is, that if we are in the simulation at `y=4mm`, but the foil is set to `foilPosition=10mm`, we only propagate the electrons 6mm instead of 10mm from their current position to the foil position. For this, it is using the simulation box position `globalOffest[1]` instead of the exact position of the single particles to not destroy the longitudinal/temporal coherence effects between the particles themselves.